### PR TITLE
Adding support for the new Turbo Drive

### PR DESCRIPTION
**Pull request by guillaumebriday**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,12 +1,20 @@
 function handleVueDestruction(vue) {
-  var turbolinksEvent = vue.$options.turbolinksDestroyEvent || 'turbolinks:visit';
+  const turbolinksEvent = vue.$options.turbolinksDestroyEvent || 'turbo:visit';
+
   document.addEventListener(turbolinksEvent, function teardown() {
     vue.$destroy();
     document.removeEventListener(turbolinksEvent, teardown);
   });
+
+  if (!vue.$options.turbolinksDestroyEvent) {
```
Comment (guillaumebriday) [inline index.js:9]: Probably not the best way to do it, but I would like to keep the support for both Turbolinks and Turbo

Comment (excid3) [reply index.js:9]: We should be able to check if `Turbo` or `Turbolinks` is defined. I know Turbolinks also has a `supported` method that might be useful as well. Not sure if Turbo has an equivalent. https://github.com/turbolinks/turbolinks/blob/master/README.md#turbolinkssupported

That would let us set the `event` dynamically to `turbo:load` or `turbolinks:load` depending on what's being used so we can continue supporting both. Same for the destroy event.

Maybe just make a helper method for each?

**Comment by excid3** [comment]
Thanks for this PR @guillaumebriday! I ended up refactoring the library to remove the "turbolinks" specific stuff inside on top of what you did. Appreciate the help! 🙏

**Comment by guillaumebriday** [comment]
@excid3 Thanks!! It's so much better now 👏

**Pull request closed without merging**